tasks/quality_control/task_screen.wdl

```diff
@@ -77,7 +77,7 @@ task check_reads {
         # First Pass; assuming average depth
         mash sketch -o test -k 31 -m 3 -r ~{read1} ~{read2} > mash-output.txt 2>&1
         grep "Estimated genome size:" mash-output.txt | \
-          awk '{if($4){printf("%d", $4)}} END {if (!NR) print "0"}' > genome_size_output
+          awk '{if($4){printf("%5.0f\n", $4)}} END {if (!NR) print "0"}' > genome_size_output
```

Comment:
May need to add to line 98 since that grabs estimated genome size from mash-output.txt, too